libtenzir/src/detail/installdirs_relocatable.cpp.in

```diff
@@ -7,6 +7,7 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 #include "tenzir/detail/assert.hpp"
+#include <string_view>
```

Comment:
The include order looks wrong here. Likely not picked up by clang-format because it's a template file for CMake?